src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/src/compiled_model/import_export.cpp

```diff
@@ -27,17 +25,36 @@ const std::vector<ov::element::Type_t> ovExecGraphInfoElemTypes = {
         ov::element::bf16,
         ov::element::boolean,
 };
-
 INSTANTIATE_TEST_SUITE_P(ov_compiled_model,
                          OVCompiledGraphImportExportTest,
                          ::testing::Combine(
                                  ::testing::ValuesIn(ovExecGraphInfoElemTypes),
                                  ::testing::Values(targetDevice),
                                  ::testing::Values(pluginConfig)),
                          OVCompiledGraphImportExportTest::getTestCaseName);
-
 INSTANTIATE_TEST_SUITE_P(
         ov_compiled_model, OVClassCompiledModelImportExportTestP,
         ::testing::Values(targetDevice));
 
-}  // namespace
+const std::vector<ov::element::Type> nPrc = {
+    ov::element::i8,
+    ov::element::i16,
+    ov::element::i32,
+    ov::element::i64,
+    ov::element::u8,
+    ov::element::u16,
+    ov::element::u32,
+    ov::element::u64,
+    ov::element::f16,
+    ov::element::f32,
+    ov::element::f64,
+    ov::element::bf16,
+};
+
+INSTANTIATE_TEST_SUITE_P(ov_compiled_model,
```

Comment:
@sbalandi FYI

Reply:
Seems there are the different data types, original tests is `ov::element::Type_t` which also is referenced by other plugins tests case, while new added test case is `ov::element::Type`. There will be compiling error if we set `ov::element::Type_t`  to `ov::element::Type_t`.
Better solution is that we adopt  `ov::element::Type` for all precision parameter, but it need also modify more places, including NPU plugin test cases.